### PR TITLE
Getting the lookupSpotifyInfo func working on OSX

### PR DESCRIPTION
**Pull request by learoyklinginsmith**

### CODE REVIEW

File: songLyrics [hunk 1]
```diff
@@ -316,14 +326,14 @@ elif [[ "$1" == 'rhythmbox' || "$1" == 'rbox' ]]; then
     exit 0;
 
 ########################################
-# look up the username from the spotify dbus api
+# look up the lyrics from the currently played spotify song
```
Comment (nick-parry) [inline songLyrics]: Playing?

File: songLyrics [hunk 2]
```diff
@@ -230,9 +230,19 @@ function do_it_azlyrics() {
...
-    TITLE="$(dbus-send --print-reply --session --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.freedesktop.DBus.Properties.Get string:'org.mpris.MediaPlayer2.Player' string:'Metadata' | grep -A 3 title | grep string | grep -v xesam | sed 's/^\s*//' | sed 's/^variant\s*//' | cut -d ' ' -f 2- | tr '(' ' ' | tr ')' ' ' | tr '"' ' ' )";
+      TITLE="$(dbus-send --print-reply --session --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.freedesktop.DBus.Properties.Get string:'org.mpris.MediaPlayer2.Player' string:'Metadata' | grep -A 3 title | grep string | grep -v xesam | sed 's/^\s*//' | sed 's/^variant\s*//' | cut -d ' ' -f 2- | tr '(' ' ' | tr ')' ' ' | tr '"' ' ' )";
+
+    elif [[ "$OSTYPE" == "darwin"* ]]; then
+      ARTIST="$(osascript -e 'tell application "Spotify" to artist of current track as string')";
+
+      TITLE="$(osascript -e 'tell application "Spotify" to name of current track as string')";
+
+    else
+      echo "Are you using windoze??"
```
Comment (nick-parry) [inline songLyrics]: As much as I want to troll the Microsoft users, maybe this should just be a more general "Your OS isn't supported" sort of message? I could take it either way I guess. Whatever your heart tells you.

**Comment by nick-parry** [comment]
Legit. It seems to work without issue on Ubuntu 14.04:

```
nick@nickp-laptop1:~/GIT/songLyrics(learoyklinginsmith-spotify_osx_fix)$ ./songLyrics spotify
Looking up title by Spotify artist and title...
Artist:  Slaves 
Title:  Death Never Lets Us Say Goodbye 
(http://www.azlyrics.com/lyrics/slaves/deathneverletsussaygoodbye.html)  Slaves  -  Death Never Lets Us Say Goodbye : 

Try to remain whole as we spiral out of control
The weight of the world takes its toll
```

Here is origin/master:

```
nick@nickp-laptop1:~/GIT/songLyrics(master)$ ./songLyrics spotify
Looking up title by Spotify artist and title...
Artist:  Slaves 
TITLE:  Death Never Lets Us Say Goodbye 
(http://www.azlyrics.com/lyrics/slaves/deathneverletsussaygoodbye.html)  Slaves  -  Death Never Lets Us Say Goodbye : 

Try to remain whole as we spiral out of control
The weight of the world takes its toll
```

**Comment by learoyklinginsmith** [comment]
Sweet. Merge/close if you want to.

**Comment by nick-parry** [comment]
Sounds good. If you don't want to fix up the comments, I can merge it now. Did you see my previous comments?

File: songLyrics [hunk 3]
```diff
@@ -230,9 +230,19 @@ function do_it_azlyrics() {
 # Sets the global ARTIST and TITLE varibles
 ################################################################################
 function lookupSpotifyInfo() {
-    ARTIST="$(dbus-send --print-reply --session --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.freedesktop.DBus.Properties.Get string:'org.mpris.MediaPlayer2.Player' string:'Metadata' | grep -A 3 artist | grep string | grep -v xesam | sed 's/^\s*//' | cut -d ' ' -f 2- | tr '(' ' ' | tr ')' ' ' | tr '"' ' ' )";
+    if [[ "$OSTYPE" == "linux-gnu" ]]; then
```
Comment (nick-parry) [inline songLyrics]: Maybe some comments here describing what is going on?

**Review by nick-parry — CHANGES_REQUESTED** [review]
Just a few things that may or may not be important.

**Comment by learoyklinginsmith** [comment]
Alright done.

**Review by nick-parry — APPROVED** [review]

**Comment by nick-parry** [comment]
Legit. Thanks.

**Pull request merged**